### PR TITLE
Update some module versions

### PR DESCRIPTION
**Pull request by pinin4fjords**
## PR checklist

Just attempting an update of a number of tools relevant to rnaseq, to improve ARM compatibility.

Also a couple of nf-test migrations

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/picard/addorreplacereadgroups/main.nf [hunk 1]
```diff
@@ -4,8 +4,8 @@ process PICARD_ADDORREPLACEREADGROUPS {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/picard:3.2.0--hdfd78af_0' :
-        'biocontainers/picard:3.2.0--hdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/picard:3.3.0--hdfd78af_0' :
+        'quay.io/biocontainers/picard:3.3.0--hdfd78af_0' }"
```
Comment (drpatelh) [inline modules/nf-core/picard/addorreplacereadgroups/main.nf:8]:
```suggestion
        'biocontainers/picard:3.3.0--hdfd78af_0' }"
```

File: modules/nf-core/picard/bedtointervallist/main.nf [hunk 2]
```diff
@@ -4,8 +4,8 @@ process PICARD_BEDTOINTERVALLIST {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/picard:3.2.0--hdfd78af_0' :
-        'biocontainers/picard:3.2.0--hdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/picard:3.3.0--hdfd78af_0' :
+        'quay.io/biocontainers/picard:3.3.0--hdfd78af_0' }"
```
Comment (drpatelh) [inline modules/nf-core/picard/bedtointervallist/main.nf:8]:
```suggestion
        'biocontainers/picard:3.3.0--hdfd78af_0' }"
```

File: modules/nf-core/picard/cleansam/main.nf [hunk 3]
```diff
@@ -4,8 +4,8 @@ process PICARD_CLEANSAM {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/picard:3.2.0--hdfd78af_0' :
-        'biocontainers/picard:3.2.0--hdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/picard:3.3.0--hdfd78af_0' :
+        'quay.io/biocontainers/picard:3.3.0--hdfd78af_0' }"
```
Comment (drpatelh) [inline modules/nf-core/picard/cleansam/main.nf:8]:
```suggestion
        'biocontainers/picard:3.3.0--hdfd78af_0' }"
```

Comment (drpatelh) [reply modules/nf-core/picard/cleansam/main.nf:8]: Need to make this change for all other modules too

**Review by JoseEspinosa — APPROVED** [review]
Awesome job! 🚀

**Comment by pinin4fjords** [comment]
Thanks @JoseEspinosa !

**Review by maxulysse — APPROVED** [review]

**Pull request merged**